crates/sui-config/src/p2p.rs

```diff
@@ -241,6 +260,23 @@ pub struct DiscoveryConfig {
     /// If unspecified, this will default to no limit.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub get_known_peers_rate_limit: Option<NonZeroU32>,
+
+    /// Access Type of this node.
+    /// If the node is Public, anyone could access it.
```

Comment:
nit, but I'd move the documentation on meaning of Public/Private to the `AccessType` enum

can you also define more explicitly what "access" means?